.github/workflows/phpstan.yml

```diff
@@ -0,0 +1,34 @@
+name: phpstan static analysis
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
```suggestion
  pull_request: ~
```

Reply:
I took inspiration of the exising job for this settings

(same for the comment below)

Reply:
It's never a shame to improve things :)